data-manager/src/main/java/org/orbisgis/datamanager/postgis/POSTGIS.java

```diff
@@ -193,4 +193,13 @@ public ISpatialTable getSpatialTable(String tableName) {
         LOGGER.error("The table '" + tableName + "' is not a spatial table.");
         return null;
     }
+    
+    @Override
+    public boolean hasTable(String tableName) {
+        try {
+            return JDBCUtilities.tableExists(connectionWrapper,tableName);
+        } catch (SQLException ex) {
+            return false;
```

Comment:
The exception should be at least logged.

Reply:
Is it necessary, because it's just an utility to test if a table exits so true or false.